openedx/core/djangoapps/profile_images/tests/test_views.py

```diff
@@ -104,6 +109,15 @@ class ProfileImageUploadTestCase(ProfileImageEndpointTestCase):
     """
     _view_name = "profile_image_upload"
 
+    def check_upload_event_emitted(self):
+        """
+        Make sure we emit a UserProfile event corresponding to the
+        profile_image_uploaded_at field changing.
+        """
+        self.assert_profile_event_emitted(
+            profile_image_uploaded_at={'old_value': u'None', 'new_value': unicode(TEST_UPLOAD_DT)}
```

Comment:
Not since the versioned URL work - see https://github.com/edx/edx-platform/pull/7517